host-controller/src/main/java/org/jboss/as/host/controller/Main.java

```diff
@@ -367,7 +367,7 @@ public static HostControllerEnvironmentWrapper determineEnvironment(String[] arg
                     if (initialHostConfig == null) {
                         return new HostControllerEnvironmentWrapper(HostControllerEnvironmentWrapper.HostControllerEnvironmentStatus.ERROR);
                     }
-                } else if (arg.startsWith(CommandLineConstants.MASTER_ADDRESS)) {
+                } else if (arg.startsWith(CommandLineConstants.PRIMARY_ADDRESS)) {
```

Comment:
Please, try to focus the changes on this PR to only messages of the loggers, that will avoid potential conflicts and keep the changes related to its issue.

Reply:
@yersan PR is updated. Thanks.